Activity.js

```diff
@@ -45,20 +45,21 @@ class Activity {
 
   markComplete() {
     this.completed = true;
-    pastActivities.push(this);
-    console.log(pastActivities);
+    // pastActivities.push(this);
+    // console.log(pastActivities);
   }
 
   saveToStorage() {
     //save to pastActivities array
     // var abc = JSON.stringify(pastActivities);
     // localStorage.setItem('a', abc);
-    localStorage.setItem('abc', JSON.stringify(pastActivities));
-    localStorage.getItem('abc', JSON.parse('abc'))
+    localStorage.setItem(`${this.id}`, JSON.stringify(this));
```

Comment:
**Option:**  You can also do `this.id.toString()`

Reply:
Good tip!